scripts/parse-vcf/README.md

```diff
@@ -0,0 +1,40 @@
+# bcftools image and annotation processing script
+
+## Environment
+
+The Dockerfile in this directory creates a bcftools image and installs some additional libraries for parsing local or remote vcf files using the `parse_vcf` python module. This parsing module is tailored to gnomAD annotation files and builds a csv comprised of the fields required by the SSMP application.
+
+To build the image, run `docker build -t bcftools .` from the directory containing the Dockerfile.
+
+The entrypoint is the bcftools binary, so you can run an arbitrary bcftools command in a local directory against a local file with something like:
+
+```bash
+docker run --rm -v `pwd`:`pwd` -w `pwd` bcftools stats <some-vcf-in-the-current-working-directory.bgz>
+```
+
+## Running the script
+
+You can use the image to run the `parse_vcf` python module, which will parse a gnomAD annotation vcf and write the filtered results to a csv that can be imported into a database.
+
+```bash
+ docker run --rm -v `pwd`:`pwd` -w `pwd` --entrypoint="python" bcftools -m parse_vcf --source gnomad_2.1.1_exome --out my-local-results-directory https://gnomad-public-us-east-1.s3.amazonaws.com/release/2.1.1/vcf/genomes/gnomad.genomes.r2.1.1.sites.vcf.bgz
+```
+
+Note that even when processing remote files, you will need to have the appropriate index file (e.g. `gnomad.genomes.r2.1.1.sites.vcf.bgz.tbi`) in the directory with the script.
+
+Though the `docker run...` approach will work, it is less than ideal given the program's current state. Because `parse_vcf` uses the python multiprocessing module to spawn child processes, these processes won't be able to print their progress/debugging messages to the host machine. It will likely be easier to run and debug the script from within the container itself, either by opening it in vscode or logging in with something like `docker run -it --rm -v `pwd`:`pwd`-w`pwd` --entrypoint="" bcftools bash`.
+
+## What the script does
+
+The parsing script will process a vcf of variant annotations, returning a subset of fields and combining them into a single csv, which can then be moved to a database. By filtering out unnecessary values, the size of the annotation file can be greatly reduced. Besides the `pos`, `chrom`, `alt`, and `ref` fields, the script will also return `nhomalt`, `AF`, `AC` and relevant `VEP` fields (if available). If you would like to adjust this output, the script will have to be manually updated (see `parse_vcf_response` function). There is also some data type coercion that is tailored to the configuration of the mongo database for ease of import that may need to be changed if the script is used for other purposes.
```

Comment:
To confirm, we are only using nhomalt, AF, and AC from gnomad right?

Reply:
Yeah -- for now we aren't using the VEP fields, as they're coming from CADD. They can either be removed at processing time by commenting out the appropriate lines in the script or excluded from the database by not uploading them into mongo. Or they could be included and simply not queried, in case they might be needed later.